sdk/test/UnitTests/AWSXRayRecorderTests.cs

```diff
@@ -1146,31 +1146,31 @@ public void SetDaemonAddress(string daemonAddress)
             {
             }
         }
-        private int PlusOneReturn(int count)
+        private static int PlusOneReturn(int count)
         {
             return count + 1;
         }
 
-        private void PlusOneNoReturn(ref int count)
+        private static void PlusOneNoReturn(ref int count)
         {
             count++;
         }
        
-        private async Task<int> PlusOneReturnAsync <TResult>(int count)
+        private static async Task<int> PlusOneReturnAsync <TResult>(int count)
         {
            await Task.FromResult<int>(count++);
            return count;
         }
 
-        private async Task PlusOneNoReturnAsync<TResult>( int count)
+        private static async Task PlusOneNoReturnAsync<TResult>( int count)
         {
             await Task.FromResult<int>(count++);
         }
 
-        private async Task<int> PlusOneReturnAsyncThrowException<TResult>(int count)
+        private static async Task<int> PlusOneReturnAsyncThrowException<TResult>(int count)
         {
             await Task.FromResult<int>(count++);
-            throw new ArgumentNullException("value");
+            throw new ArgumentNullException(null, "value");
```

Comment:
Why do we need to pass a `null` here?

Reply:
This overload takes in `public ArgumentNullException(string paramName, string message)`.  An ArgumentException should always take in the parameter causing the exception as it's first parameter.  Since "value" isn't an argument, we get a violation.  We can pass in `null` for the parameter to avoid the violation, since it won't indicate a parameter that doesn't exist.  Since this is test code, I don't know if you would find value in suppressing in this case, or just passing null.  I did remove "value" as the message, since it isn't needed.

Reply:
Cool. I think this is fine.